packages/shared/ReactFeatureFlags.js

```diff
@@ -39,6 +39,9 @@ export const replayFailedUnitOfWorkWithInvokeGuardedCallback = __DEV__;
 // Warn about deprecated, async-unsafe lifecycles; relates to RFC #6:
 export const warnAboutDeprecatedLifecycles = false;
 
+// Warn about legacy context APIs
+export const warnAboutLegacyContextAPIs = false;
```

Comment:
This new feature flag should be added to the other *FeatureFlag files as well. Flow should warn about this.

Reply:
Just have added to other files. Now running `yarn flow` gives out 0 errors 😄